specutils/__init__.py

```diff
@@ -34,3 +35,16 @@ class UnsupportedPythonError(Exception):
     _load_user_io()
 
 __citation__ = 'https://doi.org/10.5281/zenodo.1421356'
+
+class Conf(_config.ConfigNamespace):
+    """
+    Configuration parameters for specutils.
+    """
+
+    always_check_continuum = _config.ConfigItem(
```

Comment:
Instead of "always", perhaps `do_continuum_function_check` or something like that?  Only because "always" could be taken as "for all functions" and it's really just "selected functions".

Reply:
changed